src/routes/+page.svelte

```diff
@@ -282,7 +285,45 @@
     </div>
 </section>
 
-<section class="flex flex-col gap-y-16 px-6 lg:px-20 py-16">
+<section
+    class="banner min-h-[633px] flex flex-col gap-y-16 justify-center items-end px-6 lg:px-20 py-16"
+>
+    <div class="flex flex-col gap-y-12">
+        <div class="flex flex-col gap-y-6">
+            <h2 class="text-6xl font-inter font-semibold tracking-tight text-white">
+                Join the Collective
+            </h2>
+            <p class="text-2xl font-interRegular text-white max-w-[732px]">
```

Comment:
`font-interRegular` shouldn't be a valid font name anymore. But as I suggested in the previous PR you shouldn't need to add `font-inter` to any other element